python/pyspark/tests/test_shuffle.py

```diff
@@ -54,6 +63,59 @@ def test_medium_dataset(self):
         self.assertTrue(m.spills >= 1)
         self.assertEqual(sum(sum(v) for k, v in m.items()), sum(range(self.N)) * 3)
 
+    def test_shuffle_data_with_multiple_locations(self):
+        # SPARK-39179: Test shuffle of data with multiple location also check
+        # shuffle locations get randomized
+
+        with tempfile.TemporaryDirectory() as tempdir1, tempfile.TemporaryDirectory() as tempdir2:
+            original = None
+            if "SPARK_LOCAL_DIRS" in os.environ:
+                original = os.environ["SPARK_LOCAL_DIRS"]
```

Comment:
you can use `os.environ.get("SPARK_LOCAL_DIRS", None)`

Reply:
done